packages/hardhat-core/src/internal/hardhat-network/provider/modules/logger.ts

```diff
@@ -27,6 +32,10 @@ export class ModulesLogger {
     this._logs.push([title, message]);
   }
 
+  public logInfo(message: string) {
+    this._logs.push(message);
+  }
+
```

Comment:
This whole `ModulesLogger` class is seriously messed up. Don't get me wrong it's none of your fault. I can't understand why they check the `enabled` flag in the `log` method and in `logWithTitle` they don't. This is probably an error and we should add this check there. I would also remove `logInfo` and simply use `log`. 

Also, the `enabled` field should be made private and `enable` renamed to `setEnabled`. Or use the Typescript get and set functions.